dietpi/dietpi-software

```diff
@@ -12759,6 +12806,41 @@ _EOF_
 
 		fi
 
+		software_id=191 # snapcast-client
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+			# Client IP needs to be added to allowed IP list
+			local snapcast_server_ip=
+			local snapcast_server_port=1704
+			local invalid_entry=
+			while :
+			do
+				if G_WHIP_INPUTBOX "${invalid_entry}Please enter the IP address of your snapcast server machine" && [[ $G_WHIP_RETURNED_VALUE =~ ^[0-9.]+$ ]]; then
+					snapcast_server_ip=$G_WHIP_RETURNED_VALUE
+					break
+				else
+					invalid_entry='[ERROR] Please enter a valid IP address. Please retry...\n\n'
+				fi
+			done
+
```

Comment:
Add `local invalid_text=` to assure it's a local variable and empty before starting the loop.